### PR TITLE
Automated Test - COMMERCE-6064 | master

### PR DESCRIPTION
**Pull request by Jamersonze**

### CODE REVIEW

**Comment by Jamersonze** [comment]
https://issues.liferay.com/browse/COMMERCE-6064, cc: @PietroBocale

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by Pietro69** [comment]
Review started

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductProductRelations.testcase [hunk 1]
```diff
@@ -48,12 +48,42 @@ definition {
...
+		CommerceCheckout.addProductRelation(
+			key_productName = "Simple T-Shirt",
+			key_productNameRelation = "Shoes");
+			
+		CommerceCheckout.productRelationSetPriority(
+			key_productRelation = "Shoes",
+			key_priorityValue = "1");
+
+		AssertElementPresent(
+			key_orderValue = "1",
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductProductRelations.testcase:84]: Please rename `key_orderValue` into `key_Value`

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by Jamersonze** [comment]
Fixed.

**Comment by kristinonias** [comment]
ci:test:sf

**Review by Pietro69 — APPROVED** [review]

**Comment by Pietro69** [comment]
ci:forward

**Pull request closed without merging**